sherpa/sim/tests/test_asymmetric.py

```diff
@@ -0,0 +1,182 @@
+#
+#  Copyright (C) 2019  Smithsonian Astrophysical Observatory
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+
+import numpy as np
+
+from sherpa.fit import Fit
+from sherpa.data import Data1DAsymmetricErrs
+from sherpa.optmethods import LevMar
+from sherpa.utils.testing import SherpaTestCase
+from sherpa.models import PowLaw1D
+from sherpa.stats import Chi2Gehrels
+from sherpa.estmethods import Covariance
+from sherpa.sim import ReSampleData
+from sherpa.astro import ui
+
+class test_sim(SherpaTestCase):
```

Comment:
This class needs the `@requires_data` decorator, since `setUp`  calls `make_path`.

`from sherpa.utils.testing import requires_data`

Also, we're moving away from using `SherpaTestCase` and moving to `py.test` style tests. I don't know if we want to make this a requirement of new tests or not @olaurino.

Reply:
I tjhink it also needs the `@requires_fits` decorator (form the same location). At least I see some of the travis tests fail with `AttributeError: module 'sherpa.astro' has no attribute 'io'` messages which make it look like this decorator is needed.